src/components/config_profile/include/config_profile/profile.h

```diff
@@ -249,6 +249,12 @@ class Profile : public protocol_handler::ProtocolHandlerSettings,
    */
   const uint32_t& default_timeout() const;
 
+  /**
+   * @brief Default timeout compensation for waiting for response to mobile app
+   * some extra time
```

Comment:
```suggestion
   * @brief Default timeout compensation for waiting some extra time for response to mobile app
```

Reply:
@ShobhitAd fixed in https://github.com/smartdevicelink/sdl_core/pull/3726/commits/55d750086399afee1b669105e487d7a4611e2dd6